src/System.Net.NetworkInformation/src/System/Net/NetworkInformation/NetworkAddressChange.OSX.cs

```diff
@@ -181,6 +181,7 @@ private static unsafe void CreateAndStartRunLoop()
                 }
             }
             s_runLoopThread = new Thread(RunLoopThreadStart);
+            s_runLoopThread.IsBackground = true;
```

Comment:
@danmosemsft had identified a couple other cases where threads should have been marked as background.  Are those being addressed in separate PRs?

Reply:
I put note on the PR. I don't think that is same case. e.g. I look at them but I do not expect code change.